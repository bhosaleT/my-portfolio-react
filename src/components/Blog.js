import React from "react";

const Blog = () => (
  <div className="blog">
    <h2 className="blog-heading">Working on that first draft</h2>
    <iframe
      title="someGIF"
      src="https://giphy.com/embed/pOZhmE42D1WrCWATLK"
      width="400"
      height="400"
      frameBorder="0"
      class="giphy-embed"
      allowFullScreen
    />
    <p>
      <a href="https://giphy.com/gifs/animation-work-job-pOZhmE42D1WrCWATLK">
        via GIPHY
      </a>
    </p>
  </div>
);

export default Blog;
