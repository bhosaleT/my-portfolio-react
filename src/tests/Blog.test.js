import ReactShallowRenderer from "react-test-renderer/shallow";
import React from "react";
import Blog from "../components/Blog";

test("The blog component should render properly", () => {
  const renderer = new ReactShallowRenderer();
  renderer.render(<Blog />);
  expect(renderer.getRenderOutput()).toMatchSnapshot();
});
