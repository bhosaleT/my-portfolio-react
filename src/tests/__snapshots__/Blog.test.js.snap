// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The blog component should render properly 1`] = `
<div
  className="blog"
>
  <h2
    className="blog-heading"
  >
    Working on that first draft
  </h2>
  <iframe
    allowFullScreen={true}
    class="giphy-embed"
    frameBorder="0"
    height="400"
    src="https://giphy.com/embed/pOZhmE42D1WrCWATLK"
    title="someGIF"
    width="400"
  />
  <p>
    <a
      href="https://giphy.com/gifs/animation-work-job-pOZhmE42D1WrCWATLK"
    >
      via GIPHY
    </a>
  </p>
</div>
`;
